test/cluster/test_truncate_with_drop.py

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import logging
+import asyncio
+
+from test.cluster.util import new_test_keyspace
+from test.pylib.manager_client import ManagerClient
+from cassandra.query import SimpleStatement, ConsistencyLevel
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_truncation_on_drop(manager: ManagerClient):
+    await manager.server_add()
+    cql = manager.get_cql()
+
+    # Create a keyspace
+    async with new_test_keyspace(manager, "with replication = {'class': 'SimpleStrategy', 'replication_factor': 1}") as ks:
```

Comment:
SimpleStrategy is deprecated, please use NetworkTopologyStrategy instead.
Ref https://github.com/scylladb/scylladb/issues/8638

How does this even pass with tablets?

Reply:
Refs also https://github.com/scylladb/scylladb/issues/25340

Reply:
fixed